Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1078 @@
+
+import requests
+import traceback
+from typing import Dict, Any
+from datetime import datetime
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+""" CONSTANTS """
+
+SECURITYSCORECARD_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+
+""" CLIENT CLASS """
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def get_portfolios(self) -> Dict[str, Any]:
+
+        return self._http_request(
+            'GET',
+            url_suffix='portfolios'
+        )
+
+    def get_companies_in_portfolio(
+        self,
+        portfolio: str,
+        grade: Optional[str],
+        industry: Optional[str],
+        vulnerability: Optional[str],
+        issue_type: Optional[str],
+        had_breach_within_last_days: Optional[int]
+    ) -> Dict[str, Any]:
+
+        request_params: Dict[str, Any] = {}
+
+        if grade:
+            request_params['grade'] = grade
+        if industry:
+            request_params['industry'] = str.upper(industry)
+        if vulnerability:
+            request_params['vulnerability'] = vulnerability
+        if issue_type:
+            request_params['issue_type'] = issue_type
+        if had_breach_within_last_days:
+            request_params['had_breach_within_last_days'] = had_breach_within_last_days
+
+        return self._http_request(
+            'GET',
+            url_suffix='portfolios/{0}/companies'.format(portfolio),
+            params=request_params,
+            error_handler=self.company_portfolio_error_handler
+        )
+
+    def get_company_score(self, domain: str) -> Dict[str, Any]:
+
+        return self._http_request(
+            'GET',
+            url_suffix='companies/{0}'.format(domain)
+        )
+
+    def get_company_factor_score(self, domain: str, severity_in: Optional[List[str]]) -> Dict[str, Any]:
+
+        request_params: Dict[str, Any] = {}
+
+        if severity_in:
+            request_params['severity_in'] = severity_in
+
+        return self._http_request(
+            'GET',
+            url_suffix='companies/{0}/factors'.format(domain),
+            params=request_params
+        )
+
+    def get_company_historical_scores(self, domain: str, _from: str, to: str, timing: str) -> Dict[str, Any]:
+
+        request_params: Dict[str, Any] = {}
+
+        if _from:
+            request_params['from'] = _from
+
+        if to:
+            request_params['to'] = to
+
+        if timing:
+            request_params['timing'] = timing
+        # API by default is set to daily
+        else:
+            request_params['timing'] = 'daily'
+
```

Comment:
Could not use `from` as argument as it's a saved Python keyword. Therefore, needed to specify it separately. 

Section changed to:
```python
request_params: Dict[str, Any] = assign_params(
            to=to,
            timing=timing,
            domain=domain
)

# assign_params cannot accept 'from' as a parameter since it's a Python keyword
if _from:
        request_params['from'] = _from
```